src/main/java/com/example/SOPSbackend/model/converter/ResponseDictionary.java

```diff
@@ -0,0 +1,64 @@
+package com.example.SOPSbackend.model.converter;
+
+import com.example.SOPSbackend.model.VaccinationEntity;
+import com.example.SOPSbackend.model.VaccinationSlotEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class ResponseDictionary {
+    public ResponseDictionary(VaccinationSlotEntity vaccinationSlotEntity, VaccinationEntity vaccination)
+    {
+        this.id = vaccinationSlotEntity.getId();
+        this.date = vaccinationSlotEntity.getDate();
+        this.vaccination = new ResponseVaccination(vaccination);
+    }
+
+    private Long id;
+    private LocalDateTime date;
+    private ResponseVaccination vaccination;
+    public static Map<String, Object> convertToMap(ResponseDictionary slot)
```

Comment:
Niezupełnie o to chodziło :P, więcej napisałem w komentarzu gdzie ta funkcja jest używana.

IMO